src/fidesops/schemas/saas/saas_config.py

```diff
@@ -113,8 +113,10 @@ def validate_request_for_pagination(cls, values: Dict[str, Any]) -> Dict[str, An
         before any field validation.
         """
 
-        # delay import to avoid cyclic-dependency error
-        from fidesops.service.pagination.pagination_strategy_factory import get_strategy
+        # delay import to avoid cyclic-dependency error - This delay doesn't work, error still happens
+        from fidesops.service.pagination.pagination_strategy_factory import (  # pylint: disable=R0401
+            get_strategy,
+        )
```

Comment:
Correct. I left the original comment on why it is imported at that location, but my addition it to note that the original intent didn't work and the circular import error is still there. Any suggestion on how to make it more clear? Or maybe just move the import back to the top since it isn't doing what was intended?

Reply:
So do you think the comment is good or should it be modified. Pylint complains here so I think we need a way to not why we are ignoring the error when the comment implies we don't?

Reply:
I mean I'd say something like `delay import to avoid cyclic-dependency import error.  We still ignore the mypy error`